src/pages/studyView/queryStudyAnalytics.ts

```diff
@@ -0,0 +1,24 @@
+import * as request from "superagent";
```

Comment:
you should put this in shared/api folder where we keep the other api clients

Reply:
The file is now moved